frontend/tests/controllers/ContestCloneTest.php

```diff
@@ -180,4 +180,37 @@ public function testCreatePrivateContestCloneWithoutAccess() {
             $this->assertSame('userNotAllowed', $e->getMessage());
         }
     }
+
+    /*
+     * Under13 users can't clone contests.
+     */
+    public function testUserUnder13CannotCloneContests() {
+        $contestData = \OmegaUp\Test\Factories\Contest::createContest();
+        ['identity' => $identity] = \OmegaUp\Test\Factories\User::createUser(
+            new \OmegaUp\Test\Factories\UserParams([
+                'birth_date' => strtotime('-10 years'),
+            ]),
+        );
+
+        // Log in the user and set the auth token in the new request
+        $login = self::login($identity);
+
+        try {
+            \OmegaUp\Controllers\Contest::apiClone(
+                new \OmegaUp\Request([
+                'auth_token' => $login->auth_token,
+                'contest_alias' => $contestData['contest']->alias,
+                'title' => \OmegaUp\Test\Utils::createRandomString(),
+                'description' => \OmegaUp\Test\Utils::createRandomString(),
+                'alias' => \OmegaUp\Test\Utils::createRandomString(),
+                'start_time' => \OmegaUp\Time::get()
+                ])
+            );
+                $this->fail(
+                    'It should not fail'
+                );
```

Comment:
same here, alignment:

```suggestion
            $this->fail('Creating contests should not have been allowed for U13');
```

also, the failure message should explain why it's failing.

same in all other places.